packages/react-components/react-popover/src/components/PopoverSurface/usePopoverSurface.ts

```diff
@@ -45,10 +46,13 @@ export const usePopoverSurface_unstable = (
       root: 'div',
     },
     root: getNativeElementProps('div', {
+      id,
+      'aria-labelledby': id,
       ref: useMergedRefs(ref, contentRef),
       role: trapFocus ? 'dialog' : 'group',
       'aria-modal': trapFocus ? true : undefined,
       ...modalAttributes,
+      tabIndex: '-1',
```

Comment:
could you add to the description why this is needed?

Reply:
Also, is this something we want to enforce? I know that for InfoButton we need the popover surface to be focusable, but is this something all popovers need?

Reply:
Tagging @smhigley to give more insight here, but screen readers need it to be able to read the popup, so it would seem to be something all popovers need.

Reply:
Ah, it's so that when there are no focusable items within the Popover, the surface itself gets focus. The original issue was that focus was remaining on the trigger when it was opened, which means the popover was fully not read out.

One other side benefit is that if someone clicks on a popover, tabbing will start from the top of the popup content, or if focus is lost it should fall to the popover root rather than document.body.

I don't believe it should cause any issues, since it won't be in the tab order. We might consider removing the focus outline too though